src/views/Generator/GeneratorTabs/ScriptPreview.tsx

```diff
@@ -55,11 +55,10 @@ export function ScriptPreview({ fileName }: ScriptPreviewProps) {
           }}
         />
       </Flex>
-      {error ? (
-        <ScriptPreviewError error={error} />
-      ) : (
-        <CodeEditor readOnly value={preview} />
-      )}
+      <CodeEditor readOnly value={preview} />
```

Comment:
The issue itself was because of monaco being remounted once you resolve the error. 
This fix isn't perfect (because the editor stays mounted and yet it's visually hidden), but it's better than what we have right now.
My other idea was to just display the error in monaco, but it looked pretty bad